docs/DxcOnUnix.rst

```diff
@@ -0,0 +1,133 @@
+======================================
+DirectXShaderCompiler on Linux & macOS
+======================================
+
+.. contents::
+   :local:
+   :depth: 3
+
+Introduction
+============
+
+DirectXShaderCompiler (DXC) is based on LLVM/Clang, which is originally
+cross-platform. However, to support HLSL, certain Windows specific techniques
+(like COM, SAL, etc.) are introduced to solve technical issues on the Windows
+platform, which also makes DXC not compilable/runnable on non-Windows platforms.
+
+Upon `several <https://github.com/Microsoft/DirectXShaderCompiler/issues/1082>`_
+requests from the community, we have started the effort to enable compilation
+and running of DirectXShaderCompiler on non-Windows platforms (Linux and macOS).
+
+Current Status
+==============
+
+Up and Running
+--------------
+We have currently reached the point where we can successfully build and run DXC
+on Linux and macOS. We are also able to run SPIR-V CodeGen tests on these
+platforms.
```

Comment:
We should mention that both SPIR-V and DXIL CodeGen works. Otherwise, this sounds like we only support SPIR-V CodeGen.

Reply:
Done